src/app/components/input.component.ts

```diff
@@ -39,6 +39,7 @@ import { HlmPDirective } from './spartan/ui-typography-helm/src/lib/hlm-p.direct
 			[attr.maxlength]="maxchar"
 			[attr.max]="max"
 			[type]="fieldType"
+			[readonly]="readonly || false"
```

Comment:
Can you elaborate what is the purpose of adding `| | false` ?

Reply:
This is a good question!